Content.Client/UserInterface/Systems/Storage/Controls/ItemGridPiece.cs

```diff
@@ -136,12 +143,12 @@ protected override void Draw(DrawingHandleScreen handle)
                 var offset = size * 2 * new Vector2(x - boundingGrid.Left, y - boundingGrid.Bottom);
                 var topLeft = PixelPosition + offset.Floored();
 
-                if (GetTexture(adjustedShape, new Vector2i(x, y), Direction.NorthEast) is {} neTexture)
```

Comment:
These formatting fixes should not be made, at least not here downstream

Reply:
I don't recall ever doing anything with formatting, but I did do some copying and pasting since I was working on this pre-RMC renaming